DeepCat.js

```diff
@@ -139,14 +139,14 @@
 	}
 
 	function getAjaxRequest( searchTerm, searchTermNum ) {
-		var categoryString = DeepCat.extractDeepCatCategory( searchTerm ),
-			userParameter = {
+		({  categoryString, depth } = DeepCat.extractDeepCatCategory( searchTerm ));
```

Comment:
@thiemowmde True. Just pass it through; optionally replace depth 0 with 'incategory'.